.github/workflows/sdk-generation.yaml

```diff
@@ -50,12 +51,9 @@ on:
       write_suggestions:
         description: "Set to 'True' if you want to receive PRs with OpenAPI suggestions"
         required: false
-        default: "false"
-        type: string
```

Comment:
Did you remove these on purpose? I believe I added them recently due to an oncall issue where the workflow files couldn't be read

Reply:
oh shoot this must've been a rebase issue then, sorry